Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -174,14 +172,15 @@
   - FOOD
   - GLASS
   - HAPPINESS
-  - MEAT
-  - HUMAN CONTACT
```

Comment:
why?

Reply:
It was replaced with "Humanoid Contact." Not really severe but it's a bit of a safety net to use humanoid in place of human.

Reply:
fair

Reply:
Meat because alphebetical order, it is readed with the other M's. human contact changed to humanoid contact to include other species